src/components/cc-zone-input/cc-zone-input.js

```diff
@@ -351,3 +351,5 @@ export class CcZoneInput extends withResizeObserver(LitElement) {
 }
 
 window.customElements.define('cc-zone-input', CcZoneInput);
+
+console.log('hey');
```

Comment:
**suggestion:** to match my personal preferences
```suggestion
console.log('yo');
```

Reply:
oups